libbeat/_meta/config/http.reference.yml.tmpl

```diff
@@ -22,3 +22,6 @@
 # Descriptor Definition Language (SDDL) to define the permission. This option cannot be used with
 # `http.user`.
 #http.named_pipe.security_descriptor:
+
+# Defines if the HTTP pprof endpoints are enabled.
+#http.pprof.enabled: false
```

Comment:
It is probably worth adding a note about the security implications of having this enabled.